docs/lms-openapi.yaml

```diff
@@ -5145,26 +5306,6 @@ paths:
         Override the list method to expire records that are past the
         policy and requested via the API before returning those records.
       parameters:
-      - name: uuid
```

Comment:
this is unrelated. Because people don't always regenerate the open API documentation, if you run `make swagger`, you will pick up other people's changes as well.